client/webpack.storybook.config.js

```diff
@@ -37,6 +37,14 @@ config.module.loaders.push(
   }
 );
 
+config.externals = {
+  jsdom: 'window',
+  cheerio: 'window',
```

Comment:
From the link above: we could possibly leave cheerio away (needs testing though)